extensions/cmDotCom/v1/validation.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from 'zod'
+
+export const FromNameValidationSchema = z
+  .union([
+    // 11 alphanumerical characters
+    z.string().nonempty({ message: 'Missing "From/sender name"' }).max(11),
```

Comment:
I think we might need to remove the `nonempty` check on line #6 and #10 because it might cause issues when people erase a previously set value in the app. Instead, in `sendSms.ts` we should check that not both values (from settings & action fields are empty).

# Example

## Timestamp 1

- value of `fromName` in settings equals `"Nick"`
- value of `fromName` in action field equals `"Michal"`

It will use Michal 👍 

## Timestamp 2

The user removes/clears the value of `fromName` in the settings via the app (by using backspace in the input field). The value will be an empty string (`""`) and not `undefined` or `null` which will result in:
- value of `fromName` in settings equals `""`
- value of `fromName` in action field equals `"Michal"`

From my understanding of reading the code, this will now throw a Zod error because `fromName` in settings equals an empty string. Is that correct?

# Instead

Remove the `nonEmpty` validation and do the check in `sendSms.ts`.

```typescript
if (
  (isNil(value.settings.fromName) || isEmpty(value.settings.fromName)) &&
  (isNil(value.fields.fromName) || isEmpty(value.fields.fromName))
) {
  ctx.addIssue({
    code: z.ZodIssueCode.custom,
    fatal: true,
    message: '"fromName" is missing in both settings and in the action field.',
  });
}
```